lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -387,12 +387,12 @@ def test_get_redirect_url(self):
         )
 
         self.assertIn(
-            '&child=first',
+            'child=first',
             get_redirect_url(self.course_key, self.section.location, child='first'),
         )
 
         self.assertIn(
-            '&child=last',
+            'child=last',
```

Comment:
Hmm.  This improves the fragility of the `&` vs `?` issue, but adds the problem that it doesn't check that `child` is the start of the param.  So if there were a `grandchild` param, this would match it now.  That's probably a theoretical concern that's not worth worrying about, since the test isn't passing in arbitrary params.